controller/editorcontroller.php

```diff
@@ -670,13 +670,18 @@ public function config($fileId, $filePath = NULL, $shareToken = NULL, $inframe =
         }
 
         if ($folderLink !== NULL && $inframe !== 2) {
-            $params["editorConfig"]["customization"]["goback"] = [
-                "url"  => $folderLink
-            ];
+            list($majorVersion) = \OC_Util::getVersion();
+            if ($majorVersion < 18) {
```

Comment:
For what?

Reply:
The OCA.Files.Sidebar API is only available on Nextcloud 18, so the callback url only needs to be left out there.

Reply:
I do not use `OCA.Files.Sidebar`

Reply:
This is the new 18 API, as used in https://github.com/ONLYOFFICE/onlyoffice-nextcloud/pull/204/files#diff-77ab163850ef0fa98d6a9ee9fb6bbcafR128

Reply:
Leave it for the current version? 
https://github.com/ONLYOFFICE/onlyoffice-nextcloud/blob/67a7b4b0c0cc8bbe5948c118084ed0b3acabf609/js/main.js#L132-L134
https://github.com/ONLYOFFICE/onlyoffice-nextcloud/blob/67a7b4b0c0cc8bbe5948c118084ed0b3acabf609/js/main.js#L158-L159